site/_docs/posts.md

```diff
@@ -111,6 +111,24 @@ Linking to a PDF for readers to download:
   </p>
 </div>
 
+## A typical post
+
+Jekyll can handle many different iterations of the idea you might associate with a "post," however a standard blog style post, including an Title, Layout, Publishing Date, and Categories might look like this:
+
+```
+---
+layout: post
+title:  "Welcome to Jekyll!"
+date:   2015-11-17 16:16:01 -0600
+categories: jekyll update
+---
+You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
```

Comment:
Updated, but this is pulled directly from the sample post in the current _posts directory in the repo.